src/lib/components/schedule/schedule-form-view.svelte

```diff
@@ -195,6 +207,10 @@
         payloads={schedule?.action?.startWorkflow?.input}
         error={errors['input']}
       />
+      <AddSearchAttributes
```

Comment:
Do we want to add this UI yet? Or should we wait until filtering by search attribute is supported?